support-frontend/assets/pages/weekly-subscription-checkout/components/thankYou.tsx

```diff
@@ -95,6 +117,29 @@ function StartDateCopy({
 	return null;
 }
 
+function StartDateCopyThreeTier({ startDate }: { startDate: Option<string> }) {
+	if (startDate) {
+		return (
+			<Text title={'When will my subscription start?'}>
+				<SansParagraph>
+					<strong>
+						Your first issue of Guardian Weekly will be published on&nbsp;
+					</strong>
+					{formatUserDate(new Date(startDate))}
+				</SansParagraph>
+				<SansParagraph>
+					<strong>Your digital benefits will start today.</strong>
+					<br />
+					Please ensure you are signed in on all your devices to enjoy unlimited
+					app access and ad-free reading.
+				</SansParagraph>
+			</Text>
+		);
+	}
+
+	return null;
```

Comment:
You could remove the logic of optional parameters to the implementing / parent component.

This would make the component a little neater and more expressive i.e. the component expresses that it needs a start date to render.

e.g.
```tsx
<Parent>
  {startDate && <StartDateCopyThreeTier startDate={startDate} />}
</Parent>
```

Reply:
Actually I dont think an optional check is required at all (unable to get an undefined start date further up). Have removed the optional checks and test run against three tier and existing GW checkout thankyou's. Updated.